slasher/detection/attestations/spanner_test.go

```diff
@@ -111,6 +119,113 @@ func TestSpanDetector_DetectSlashingForValidator(t *testing.T) {
 	}
 }
 
+func TestSpanDetector_DetectSlashingForValidator_MultipleValidators(t *testing.T) {
```

Comment:
in the old tests i have a longer test following proto diagram to see if our implementation updated spans as expected on a more complex scenario. can you add it back?